### ISSUE TITLE
problem with swap and typo in /etc/udev/scripts/grml-udev-rebuildfstab

### ISSUE THREAD

**Issue by felix-egli** [grml/grml#199]
**Description of the bug**
I found a problem with grml 2022.11:
I have installed grml on a USB-stick with persistence. The problem I have is, that after I booted a computer from this USB-stick, which has a swap partition configured, grml tries to enable this swap partition on all the future boots from that USB-stick, which is not possible and ends in not booting up, but waiting on this swap enable forever.

A solution for this could be, to remove the added entries from fstab in an early stage on startup, because they might not be valid and are recreated anyway.
Another solution could be, to not add swap entries to fstab.

I tried to not create entries in fstab by setting CONFIG_FSTAB='no' in /etc/grml/autoconfig, which did not work, because of a typo (missing 'o'):

```sh
# diff -u grml-udev-rebuildfstab.dist grml-udev-rebuildfstab
--- grml-udev-rebuildfstab.dist    2022-12-17 08:54:05.165267587 +0100
+++ grml-udev-rebuildfstab    2022-12-17 08:54:21.773545150 +0100
@@ -32,7 +32,7 @@
 umask 022
 
 
-[ -e /etc/grml/autconfig ] && . /etc/grml/autoconfig
+[ -e /etc/grml/autoconfig ] && . /etc/grml/autoconfig
 
 if ! [ -w "$(dirname $MNTFILE)" ] ; then
    echo "$0: can not write to $MNTFILE">&2

```

**Steps To Reproduce**
- create a USB-stick with persistence.
- boot with it a system which has a swap partition.
- boot another system with the same USB-stick.

**Expected behavior**
I can always boot any system, regardless of which systems were booted from the USB-stick before.

**In which Grml Version did the bug occur?**
2022.11

**Comment by mika** [comment]
@felix-egli thanks for the bug report and fix! (And sorry for the late reply :-/)

Regarding getting swap enabled: this definitely sounds like a bug, though AFAICT swap shouldn't be enabled by default at all, hmmmmmm :) Booting with boot option "noswap" doesn't change anything for you either, nor?

**Comment by felix-egli** [comment]
Hi 

Sorry for my late answer. I created a new usb-stick with persistence, and swap was not activated. 
Then I installed additional software and updated the packages, and afterwerds swap was activated automatically 
Maybe updating the packages is something which should not be done in grml. 

noswap did not help. 

I was able to prevent the automatic swap activation by deleting '/usr/lib/systemd/system/swap.target'. 
I'm still able to activate swap with grml-swapon, but I don't need it. 

Regards, Felix
